examples/systemd/sysconfig.node_exporter

```diff
@@ -0,0 +1 @@
+OPTIONS="-collector.textfile.directory /var/lib/node_exporter/textfile"
```

Comment:
This is going to vary based on the system and distro, which is why it's very difficult to provide example files.